graphite_api/functions.py

```diff
@@ -661,6 +661,43 @@ def asPercent(requestContext, seriesList, total=None):
     return resultList
 
 
+def divideSeriesLists(requestContext, dividendSeriesList, divisorSeriesList):
+    """
+    Iterates over a two lists and divides list1[0] by list2[0], list1[1] by list2[1] and so on.
+    The lists need to be the same length
+    """
+
+    if len(dividendSeriesList) != len(divisorSeriesList):
+        raise ValueError("dividendSeriesList and divisorSeriesList argument must have equal length")
+
+    results = []
+
+    #for dividendSeries in dividendSeriesList:
+    for i in range(0, len(dividendSeriesList)):
+        dividendSeries = dividendSeriesList[i]
+        divisorSeries = divisorSeriesList[i]
+
+        name = "divideSeries(%s,%s)" % (dividendSeries.name, divisorSeries.name)
+        bothSeries = (dividendSeries, divisorSeries)
+        step = reduce(lcm,[s.step for s in bothSeries])
+
+        for s in bothSeries:
+            s.consolidate( step // s.step )
+
+        start = min([s.start for s in bothSeries])
+        end = max([s.end for s in bothSeries])
+        end -= (end - start) % step
+
+	#toto
+	#raise ValueError("%s" % repr(zip(*bothSeries)))
```

Comment:
What's this?

Reply:
I forgot to clean up my tests before committing :)